packages/dashboard/src/routes/projects/[projectId]/machine-translations.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+
+    import {Button, Link, Tab, TabContent, Tabs, Toggle} from "carbon-components-svelte";
+    import {mockOrganization, mockProject} from "$lib/mockData";
+
+    let toggled = true; // TODO: Save the machine translations toggle locally
+</script>
+
+<h1>Machine Translations</h1>
+<p>Translations can be autogenerated through <Link href="https://www.deepl.com/">Deepl</Link> for increased productivity of translations.</p>
+
+<p><Link href="https://support.deepl.com/hc/en-us/articles/360019925219-Languages-included-in-DeepL-Pro">Deepl supported languages can be seen here.</Link></p>
+
+<Tabs>
+    <Tab label="Usage"/>
+    <Tab label="Settings"/>
+    <div slot="content">
+        <TabContent>
+            <div>
+                <h2>Project Usage:</h2>
+                <p>Characters used for machine translations in current project.</p>
+                <p>Project machine translations: {mockProject.machineTranslations}</p>
+            </div>
+            <div>
+                <h2>Organization Usage:</h2>
+                <p>Characters used for machine translations by all projects in the organization.</p>
+                <p>Organization machine translations: {mockOrganization.machineTranslations}</p>
+            </div>
+        </TabContent>
+        <TabContent>
+            <div class="toggle">
+                <Toggle labelText="Machine Translations" labelA="Disabled" labelB="Enabled" bind:toggled />
+            </div>
+            <div class="toggle">
+                <Toggle labelText="Automatically translate new keys" disabled={!toggled}/>
+            </div>
+            <div class="toggle">
+                <Toggle labelText="Automatically translate new languages" disabled={!toggled}/>
+            </div>
+        </TabContent>
+    </div>
+</Tabs>
+
+<style>
```

Comment:
Please do not use custom CSS styles instead use TailwindCSS. [Here](https://tailwindcss.com/docs/margin#class-reference) is the documentation for margins ([which should never by `px` btw ](https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/#:~:text=px%20is%20not%20scalable%2C%20it,px%20unit%20by%20the%20browser.)). 

So `<div class="toggle">` becomes `<div class="mb-2">`. Even better, most of the times Carbon components allow to pass classes down, so `<Toggle class="mb-2" ...>` likely works